model/task/task.go

```diff
@@ -2807,14 +2807,6 @@ func GetTasksByVersion(versionID string, sortBy []TasksSortOrder, statuses []str
 		})
 	}
 
-	// expand execution tasks in display tasks
-	pipeline = append(pipeline, bson.M{"$lookup": bson.M{
-		"from":         Collection,
-		"localField":   ExecutionTasksKey,
-		"foreignField": IdKey,
-		"as":           ExecutionTasksFullKey,
-	}})
```

Comment:
This was removed in favor of performing this expansion in the task resolver. This is an expensive operation and we dont always use these fields. By performing it in the task resolver we only do this operation if we specifically request the `executionTasksFull` field